server/main.go

```diff
@@ -107,11 +107,11 @@ func Server(port int, service interfaces.IService, memoryRepository interfaces.I
 	// CHOOSE TO USE POSTRGRES OR IN_MEMORY IMPLEMENTATION BY COMMENTING / UNCOMMENTING
 
 	// ** USE LOCAL POSTGRES DB **
-	oauthRepository := oauthRepo.NewOauthRepository(config.DB)
-	oauthService := &oauthSvc.Service{Repo: oauthRepository}
+	// oauthRepository := oauthRepo.NewOauthRepository(config.DB)
+	// oauthService := &oauthSvc.Service{Repo: oauthRepository}
 
 	// ** USE THE IN MEMORY IMPLEMENTATION **
-	// oauthService := &oauthSvc.Service{Repo: memoryRepository}
+	oauthService := &oauthSvc.Service{Repo: memoryRepository}
```

Comment:
I think we can leave the postgres db repository implementation as the default. WDYT?

Reply:
We only change that when we are deploying the app to either Lightsail or EC2, the reason here to keep the memoryRepository is because when someone clones, and wants to use `make run_server_local` directly, they can do that without changing anything or making .env, so for the ease of use, I think this one is better.